network/dag/dag.go

```diff
@@ -260,7 +262,9 @@ func (d dag) setNumberOfTransactions(tx stoabs.WriteTx, count uint64) error {
 func (d dag) getHighestClockValue(tx stoabs.ReadTx) uint32 {
 	value, err := tx.GetShelfReader(metadataShelf).Get(stoabs.BytesKey(highestClockValue))
 	if err != nil {
-		log.Logger().Errorf("Unable to retrieve highest LC value: %v", err)
+		log.Logger().
+			WithError(err).
+			Error("Unable to retrieve highest LC value")
```

Comment:
is LC value the same as the `clock shelf` used below?

Reply:
?